packages/shared/src/components/Sidebar.tsx

```diff
@@ -2,6 +2,7 @@ import React, { ReactElement, ReactNode, useContext, useState } from 'react';
 import Link from 'next/link';
 import SettingsContext from '../contexts/SettingsContext';
 import { Button } from './buttons/Button';
+import { BlueDot } from './notifs';
```

Comment:
Thinking maybe we should rename this to AlertDot and have the option to define a color?
Just thinking as this will become more a generic component it might be worthwhile changing it now.

Reply:
I was thinking about that as well. Though I think it would be hard to have a clear definition of the alerts since let's say, for example, the blue one, there can be multiple shades of blue and for instance colors other than `fill` (current blue) will be used.

I think it would be best for @tsahimatsliah to define this and create a guideline if necessary.

Reply:
Or why not we use the ones for status since we already have those defined.